packaging/googet/install.ps1

```diff
@@ -30,7 +30,7 @@ try
     {
         New-Service -DisplayName 'Google Cloud Operations OpenTelemetry Collector' `
             -Name 'google-cloudops-opentelemetry-collector' `
-            -BinaryPathName '%ProgramFiles%\Google\Cloud Operations\OpenTelemetry Collector\opentelemetry-collector.exe --config="C:\Users\jbebb\go\src\go.opentelemetry.io\collector\examples\otel-local-config.yaml"' `
+            -BinaryPathName '%ProgramFiles%\Google\Cloud Operations\OpenTelemetry Collector\opentelemetry-collector.exe --config="%ProgramFiles%\Google\Cloud Operations\OpenTelemetry Collector\config.yaml"' `
```

Comment:
Is it possible to extract the path to the collector installation dir to a variable?

Reply:
Alternatively, the `--config` flag may support relative path

Reply:
iirc, the issue with relative paths is that because this is invoked through Windows Service Manager the current dir is not what you would expect.